lib/commands/generate-test-static-scenario.ts

```diff
@@ -41,7 +41,7 @@ export async function handler(argv: yargs.Arguments): Promise<void> {
   console.log("input-file:");
   console.log(swaggerFilePaths);
 
-  const generator = StaticTestScenarioGenerator.create({
```

Comment:
Maybe we should wait for final product name?

Reply:
We'd like to make all the required breaking changes in one PR.